lib/labels_web/controllers/page_controller.ex

```diff
@@ -130,5 +146,14 @@ defmodule LabelsWeb.PageController do
     end
   end
 
+  defp form_values(opts \\ []) do
```

Comment:
My first thought was to create a new Ecto Schema for the form with virtual fields as we don't want to save directly the form fields to Postgres. Then using a changeset to display the value to the form, however it seems a bit too complicated and ended up just using a keyword list for the form values that we pass to the view.